### PR TITLE
Add new niceoppai extention

### PR DESCRIPTION
**Pull request by ponekub083**
Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [ ] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [ ] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

File: src/th/niceoppai/build.gradle [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+
+ext {
+    extName = 'Niceoppai'
+    pkgNameSuffix = 'th.niceoppai'
+    extClass = '.Niceoppai'
+    extVersionCode = 1
```
Comment (vetleledaal) [inline src/th/niceoppai/build.gradle:8]:
```suggestion
    extVersionCode = 25
```
... to continue from the Madara version

**Review by vetleledaal — CHANGES_REQUESTED** [review]
Since you're moving from Madara you should remove the other resources (`./multisrc/overrides/madara/niceoppai/`), and the line in `MadaraGenerator.kt`

**Comment by ponekub083** [comment]
@alessandrojean please review code and build extention for release to main apps.

File: src/th/niceoppai/build.gradle (hunk 1 above)
Comment (alessandrojean) [reply src/th/niceoppai/build.gradle:8]: It's the same package name and classname so it should appear as an update. Please apply the changes suggested by Vetle.

**Comment by alessandrojean** [comment]
The code is fine, just need to apply the changes suggested by Vetle so the new extension appears as an update to existing users.

File: src/th/niceoppai/build.gradle (hunk 1 above)
Comment (ponekub083) [reply src/th/niceoppai/build.gradle]: @alessandrojean @vetleledaal  thank you. it applied.

**Review by vetleledaal — APPROVED** [review]

**Pull request merged**